com_lscache/admin/language/en-GB/en-GB.com_lscache.ini

```diff
@@ -1,11 +1,11 @@
 COM_LSCACHE_TOOLBAR_TITLE="LiteSpeed Cache Settings (ESI Module Settings)"
-COM_LSCACHE_MODULE_CACHE_TYPE = "Cache behavior after Moudle changed"
+COM_LSCACHE_MODULE_CACHE_TYPE = "Cache Behavior After Module is Changed"
```

Comment:
Why uppercase all words like this?

Reply:
I thought it was a label. Labels are capitalized. Is it a description? (I couldn't find it in the interface to check it)